src/libcore/tests/test_xml.py

```diff
@@ -298,3 +298,166 @@ def test20_upgrade_tree(variant_scalar_rgb):
                                <float name="intIOR" value="1.33"/>
                            </bsdf>
                        </scene>""")
+
+@fresolver_append_path
+def test21_xml_save_plugin(variant_scalar_rgb):
+    from mitsuba.core import xml
+    from mitsuba.python.dict_to_xml import WriteXML
+    import os
+    abs_path = os.path.abspath('resources/data/scenes/dict/dict.xml')
+    writer = WriteXML(abs_path)
+    scene_dict = {
+            "type": "sphere",
+            "center" : [0, 0, -10],
+            "radius" : 10.0,
+        }
+    writer.configure(scene_dict)
```

Comment:
I would rather have a simpler interface, instead of the `WriteXML` class:
```python
abs_path = ...
scene_dict = ...
xml.dict_to_xml(scene_dict, abs_path)
```
What do you think?

Reply:
Also this way the `import` statement would look like:
```python
from mitsuba.python.xml import dict_to_xml
```

Reply:
Encapsulating this in a class is quite helpful, as there are numerous operations to do, and buffers to store (for indentation for example). Do you want to change the implementation itself, or simply wrap it in a `dict_to_xml` function that instantiates a `WriteXML` ?

```python
def dict_to_xml(scene_dict, abs_path):
    writer = WriteXML(abs_path)
    writer.configure(scene_dict)
```

Reply:
This looks much better!
Also should we rename the method `configure` maybe?